public/openapi/v1.0-pre.yaml

```diff
@@ -3798,8 +3764,9 @@ paths:
                 provider_trainee_id: 99157234/2/01
                 itt_start_date: '2023-01-01'
                 nationality: FR
+                course_age_range: '13914'
                 lead_partner_urn: '900020'
-                employing_school_urn: '865860'
+                employing_school_urn: '670086'
                 trn: '567899'
                 ethnicity:
                 ethnic_group: mixed_ethnic_group
```

Comment:
Looks like we still have `ethnic_group` and `ethnic_background` in the PUT request body as well, which I think we want to remove.